### ISSUE TITLE
Font Size can be set to less than 0.  Program then fails to start.

### ISSUE THREAD

**Issue by maxcheese3** [arduino/Arduino#6717]
I work at the IT desk at my school, and a student had managed to zoom out so far in the Arduino IDE that he couldn't see any of his code.  It also wouldn't let him zoom back in.  He tried restarting the IDE and it wouldn't load and the splash-screen just would disapear.  He tried system restore and that didn't fix it either.  I ran the arduino_debug.exe and saw that Java was complaining the font size was less than 0, and crashing.

I navigated to the preferences.txt file in C:\Users\user\Documents\Arduino and the font size was set to -13.  I navigated to the preferences.txt file in C:\Users\user\AppData\Local\Arduino15 and the font size was set to -37...  After setting both of these to 11, the Arduino IDE loaded succesfully.

So basically a user can zoom out to a negative font size, which causes the IDE to not be able to launch next time.

**Comment by facchinm** [comment]
Hi @maxcheese3 ,
this issue has already been addressed by https://github.com/arduino/Arduino/commit/7c27d09c6f35d926b4f0b6077588fb9c28978459 and the fixed IDE is available in nightly.
Closing as duplicate of https://github.com/arduino/Arduino/issues/6689
